components/dashboard/src/feedback-form/FeedbackComponent.tsx

```diff
@@ -83,25 +122,26 @@ function FeedbackComponent(props: { onClose: () => void; onSubmit: () => void; i
                             </p>
                         </div>
                     </div>
-                    <div className="flex justify-end mt-6">
-                        <button className="secondary" onClick={props.onClose}>
+                    <div className={"flex justify-end mt-6 " + (props.isError ? "mb-6" : "")}>
+                        <button className="secondary" onClick={onClose}>
                             Cancel
                         </button>
                         <button className="ml-2" onClick={onSubmit}>
                             Send Feedback
```

Comment:
**issue**: Showing another primary action button so close to another primary action in some cases could be confusing to users, especially since the current layout and design does not group the new action button with the feedback elements.

Reply:
Now grouped 👍🏼